controllers/module_reconciler.go

```diff
@@ -280,6 +324,10 @@ func (r *ModuleReconciler) handleBuild(ctx context.Context,
 
 	logger.Info("Image not pull-able; building in-cluster")
 
+	if err := r.injectDTKToBuildIfNeeded(ctx, kernelVersion, km.Build); err != nil {
```

Comment:
WDYT about moving the injectDTKToBuildIfNeeded into build package, where it is going to be called by in the Sync API?

Reply:
Done.